app/src/main/java/org/simple/clinic/summary/teleconsultation/contactdoctor/ContactDoctorModel.kt

```diff
@@ -3,14 +3,17 @@ package org.simple.clinic.summary.teleconsultation.contactdoctor
 import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
 import org.simple.clinic.summary.teleconsultation.sync.MedicalOfficer
+import java.util.UUID
 
 @Parcelize
 data class ContactDoctorModel(
+    val patientUuid: UUID,
```

Comment:
why do we want to track patientUuid in `ContactDoctorModel` ?

Reply:
we need `patientUuid` for loading the patient teleconsult info. You can check this [commit](https://github.com/simpledotorg/simple-android/pull/1869/commits/2c1f5130d788766a225c1d0fc618e4163d51a6ec)